src/routes/posts.js

```diff
@@ -33,7 +33,33 @@ const createPost = async (req, res, next) => {
     }
 }
 
+// GET ALL ARTICLES
+
+const fetchPosts = async (req, res, next) => {
+    try {
+        const allArticles = await db.query('SELECT * FROM posts')
+        const articles = allArticles.rows
+
+        res.status(201).json({
+            status: 'success',
+            data: articles.map((article) => ({
+                id: article.id,
+                userId: article.userId,
+                title: article.title,
+                content: article.content,
+                image: article.image,
+                published: article.published,
+                createdAt: article.createdAt,
+            })),
+        })
+    } catch (err) {
+        console.error(err.message)
+        next(err)
+    }
+}
+
 // Routes
 router.route('/').post(createPost)
+router.route('/').get(fetchPosts)
```

Comment:
so this works but just for learning purposes, mostly the reason to use `route` is to chain different HTTP methods so we don't have to repeat the same route on different lines. 

Else the usual thing is just to do `router.post('/', createPost)` if you will not use the `/` route for any other thing 
```suggestion
router.route('/')
    .post(createPost)
    .get(fetchPosts)

```

Reply:
from express documentation since i couldn't get a direct link to that specific subtopic


![Screenshot from 2022-08-18 18-35-44](https://user-images.githubusercontent.com/31732834/185469199-e8a34839-5671-49de-a9fe-9b33ec0a77da.png)

Reply:
i have made this change as well, however linting makes it a one-liner

Reply:
no worries, thanks